logos/logos-orchestrator/src/logos/capacity/capacity_planner.py

```
@@ -2295,6 +2305,13 @@ def _pick_cold_load_placement(
         if len(all_gpu_ids) < tp:
             return None  # Not enough GPUs
 
+        # vLLM reserves at least VLLM_GMU_FLOOR × GPU-total per rank regardless of
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)